### ISSUE TITLE
Broken links in documentation

### ISSUE THREAD

**Issue by marshallmcdonnell** [MLMI2-CSSI/foundry#375]
NOTE: This is related to JOSS review https://github.com/openjournals/joss-reviews/issues/5467

**Describe the bug**
Broken links in docs throughout code base:

1. Following link is broken in the docs for publishing datasets:
https://github.com/MLMI2-CSSI/foundry/blob/main/docs/publishing/publishing-datasets.md?plain=1#L11

This line has the following link that does not exist:
```
[Skip to the publication example notebook.](https://github.com/MLMI2-CSSI/foundry/blob/master/examples/foundry_publication_example.ipynb)
```

2. Broken link for contributing guide:
https://github.com/MLMI2-CSSI/foundry/blob/main/docs/how-to-contribute/contributing.md?plain=1#L34

This line has the following link that does not exist:
```
* Tests should follow [testing best practices](https://www..org/community/contribute/tests)
```

**To Reproduce**
Try to navigate to these links.

**Expected behavior**
For (1), I expect that it is now a new name for the notebook and probably found here now?
https://github.com/MLMI2-CSSI/foundry/tree/main/examples/publishing-guides

**Comment by ascourtas** [comment]
Based on clicking around, it appears that the links in the [actual GitBook documentation](https://ai-materials-and-chemistry.gitbook.io/foundry/) are not broken, but rather the `docs` hosted in the repo are significantly out of synch with the GitBook docs, thus making it confusing for the user. GitBook used to automatically synch, but in one of their updates that stopped working.

I'm going to reframe this issue to do the following:

- [ ] Re-synch GitBook with GitHub to update the `docs` md files
- [ ] Double check to make sure the GitBook links work 
- [ ] Ensure GitBook docs reflect current functionality 

In future work, we may want to consider moving away from GitBook

**Comment by ascourtas** [comment]
The GitBook - GitHub Integration has been fixed (thank you @blaiszik) and the `/docs` folder now reflects that. From here on all changes made in our [documentation](https://ai-materials-and-chemistry.gitbook.io/foundry/) will be reflected in `/docs`. Changes can be viewed in [commit 0fa7f19](https://github.com/MLMI2-CSSI/foundry/commit/0fa7f1950543a14b756b5e3f6bc21c608acb91c9) 

The links to testing best practices were indeed broken in the current documentation, and have now been fixed and deployed in [commit 68c8fbf](https://github.com/MLMI2-CSSI/foundry/commit/68c8fbfcda620a5b39b9f22b01b22d13dc2a6797)

The link to our documentation is currently available in the GitHub repo's README, but please let us know if you think it needs to be more clearly communicated. 

I will close this issue now that's resolved -- please feel free to reopen it if this solution didn't address all parts of the issue
